src/test/java/io/strimzi/test/container/StrimziKafkaContainerMockTest.java

```diff
@@ -18,8 +18,8 @@
 import static org.hamcrest.CoreMatchers.containsString;
 import static org.hamcrest.CoreMatchers.is;
 import static org.hamcrest.MatcherAssert.assertThat;
-import static org.junit.Assert.assertTrue;
```

Comment:
This is the junit 4 dependency brought via testcontainers. We should instead use `org.junit.jupiter.api.Assertions` which is junit 5 (the declared dependency in pom.xml).